pattern-hues.lic

```diff
@@ -56,14 +55,14 @@ class PatternHues
       /You should stop practicing/,
       /You find it hard to concentrate on anything/
     ]
-    scripts_to_unpause = DRC.smart_pause_all
-    bput("prep cantrip pattern hues", "You are now prepared to cast the Pattern Hues cantrip.","You already have", "You should stop practicing")
+    pattern_hues_pause_list ||= DRC.safe_pause_list
```

Comment:
```suggestion    
    pattern_hues_pause_list = DRC.safe_unpause_list
   
    unless pattern_hues_pause_list
      echo("Cannot pause, trying again shortly.")
      return
    end
```

Reply:
This is where I meant to make that suggestion to.

Reply:
You mean safe_pause_list here.

Reply:
Need ||= or an error occurs:
```
2022-08-20 01:23:30 -0400:--- Lich: error: wrong number of arguments (given 0, expected 1)
2022-08-20 01:23:30 -0400:	common:784:in `safe_unpause_list'
2022-08-20 01:23:30 -0400:	pattern-hues:69:in `cast_pattern_hues'
```

Reply:
Test it out. I am just writing from memory. :) Just pointing out you need to check return values.

Reply:
Changing name from pattern_hues.... to scripts_to_unpause.

Reply:
> 

This error is because I had suggested the wrong method. DRC.safe_pause_list has no arguments.